notification/notification-sdk/src/main/java/org/sunbird/notification/email/service/impl/SmtpEMailServiceImpl.java

```diff
@@ -28,19 +28,24 @@ public boolean sendEmail(EmailRequest emailReq) {
       logger.info("Email request is null or empty:");
       return false;
       // either email object has bcc or to list size more than 1 then pass it as bcc.
-    } else if (CollectionUtils.isNotEmpty(emailReq.getBcc()) || emailReq.getTo().size() > 1) {
+    } else if (CollectionUtils.isNotEmpty(emailReq.getBcc()) && !emailReq.getTo().isEmpty()) {
```

Comment:
This is not proper approach... 
the existing code allowed to send email when bcc is not empty OR to address is not empty... 
but latest code allows only when the bbc is not empty and to address is not empty... -- what will happen to existing implementation -- does all the cases, we have bcc and to available ??

Reply:
This code has been removed Karthik